examples/12_ammonia_synloop/tech_inputs/greenheart_config_onshore_tx.yaml

```diff
@@ -0,0 +1,307 @@
+site:
```

Comment:
Is this file here on purpose? It is in the older GreenHEART format so I think it might've been added unexpectedly